.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.4.3
```

Comment:
I'm 👍 with this, but I also saw you can define it in the Gemfile which is cool: https://rvm.io/workflow/projects#supported-files

Reply:
Just tried this and got:

```
RVM used your Gemfile for selecting Ruby, it is all fine - Heroku does that too,
you can ignore these warnings with 'rvm rvmrc warning ignore ./Gemfile'.
To ignore the warning for all files run 'rvm rvmrc warning ignore allGemfiles'.

Using /home/admin/.rvm/gems/ruby-2.4.3
```

Which means I think we'd need to update the rvm config on the cpair image if we wanted to use the Gemfile instead.